src/collection/amd/amd_gpu_marketing.rs

```diff
@@ -450,7 +448,7 @@ pub const AMD_GPU_MARKETING_NAME: &[(u32, u32, &str)] = &[
     (0x7312, 0x00, "AMD Radeon Pro W5700"),
     (0x7319, 0x40, "AMD Radeon Pro 5700 XT"),
     (0x731E, 0xC7, "AMD Radeon RX 5700B"),
-    (0x731F, 0xC0, "AMD Radeon RX 5700 XT 50th Anniversary"),
+    (0x731F, 0xC0, "AMD Radeon RX 5700 XT 50th"),
```

Comment:
I guess this one is a bit weird to truncate like this 🤔

Reply:
Yeah, you're right. Reverting that one.